syntax="proto3";

package game;

service Game{

    rpc move();

    rpc missile_approaching(missile_info) returns (missile_info);

    rpc status() returns ();

    rpc was_hit() return ();


}
message void{}

message movement_info{
    int32 Soldier_id=1;
    int32 x=2;
    int32 y=3;

}



message missile_info{
    int32 x=1;
    int32 y=2;
    int32 time=3;
    string type=4;

}
